test/inductor/test_flex_decoding.py

```diff
@@ -51,17 +51,16 @@ def create_block_mask_test(score_mod, query, key):
     return block_mask
 
 
-if HAS_GPU:
-    if HAS_CUDA:
-        test_dtypes = (
-            [torch.float16, torch.bfloat16, torch.float32]
-            if PLATFORM_SUPPORTS_BF16
-            else [torch.float16, torch.float32]
-        )
-        test_dtypes_fast = [torch.float16]
-    elif HAS_XPU:
-        test_dtypes = [torch.float32, torch.bfloat16, torch.float16]
-        test_dtypes_fast = [torch.float16]
+if GPU_TYPE == "cuda":
```

Comment:
To fix: `NameError: name 'test_dtypes_fast' is not defined`. https://github.com/pytorch/pytorch/actions/runs/14382267548/job/40339987896?pr=151012#step:22:701

Reply:
@etaf could you restart CI?